helix-front/package.json

```diff
@@ -40,18 +41,18 @@
     "Chrome > 60"
   ],
   "dependencies": {
-    "@angular/animations": "^13.3.9",
-    "@angular/cdk": "^13.3.7",
-    "@angular/common": "^13.3.9",
-    "@angular/compiler": "^13.3.9",
-    "@angular/core": "^13.3.9",
-    "@angular/flex-layout": "^13.0.0-beta.38",
-    "@angular/forms": "^13.3.9",
-    "@angular/material": "^13.3.7",
-    "@angular/platform-browser": "^13.3.9",
-    "@angular/platform-browser-dynamic": "^13.3.9",
-    "@angular/platform-server": "^13.3.9",
-    "@angular/router": "^13.3.9",
+    "@angular/animations": "13.3.9",
+    "@angular/cdk": "13.3.9",
+    "@angular/common": "13.3.9",
+    "@angular/compiler": "13.3.9",
+    "@angular/core": "13.3.9",
+    "@angular/flex-layout": "13.0.0-beta.38",
+    "@angular/forms": "13.3.9",
+    "@angular/material": "13.3.9",
+    "@angular/platform-browser": "13.3.9",
+    "@angular/platform-browser-dynamic": "13.3.9",
+    "@angular/platform-server": "13.3.9",
+    "@angular/router": "13.3.9",
```

Comment:
Caret version produced version mismatch warnings in the browser console during development, so I went back to pinning exact, consistent package versions for all `@angular` dependencies.